app/assets/javascripts/discourse/components/text-field.js.es6

```diff
@@ -1,7 +1,35 @@
 import computed from "ember-addons/ember-computed-decorators";
+import { siteDir } from "discourse/lib/text-direction";
```

Comment:
You can import all three on one line:

`import { siteDir, isRTL, isLTR } from ...`